inst/dev-examples/convert-psn-to-bbr.R

```diff
@@ -54,7 +54,10 @@ mod_sum %>% get_theta()
 # one of the nice things about bbr is how it integrates with other MeRGE packages
 # here we show how to make simple diagnositic plots with the pmplots package
 #   see more here: https://merge.metrumrg.com/expo/expo1-nonmem-foce/posts/intro-to-pmplots.html
-if (!nzchar(packageVersion("pmplots"))) remotes::install_github("metrumresearchgroup/pmplots")
+if(require(pmplots)){
```

Comment:
Using `require()` is fine, but you have the logic backwards. This is saying to install _if_ it's already installed.

You want the following (without my comments)

```
if (!require(pmplots)) { # load the library, if not installed `!require` will return TRUE
  remotes::install_github("metrumresearchgroup/pmplots") # install
  library(pmplots) # load the library, now that it's installed
}
```

Reply:
Ahh good catch. Yeah I found that `nzchar` returned an error for me. Didnt look into it too much, but with this fix I think it will be more reliable